### ISSUE TITLE
Keyboard not accessible for mobile users

### ISSUE THREAD

**Issue by Sherlybelly** [m-ades/frontend-logic#26]
## 🐛 The Bug

Screen readers on mobile devices not able to access the keyboard. 

## 🔁 Steps to Reproduce

1. Go to Chapter 7 problems on mobile
2. Go to line with text field
3. Double-tap on text field so keyboard pops up
4. Scroll right to try to get to keyboard 
5. See the error: the keyboard disappears when users swipe to the delete line button, double tap on the lines/type text box, or the rule dropdown.

## 🎯 Expected Behavior

On slack accessibility feed-back

## 💻 Environment
Tested on: 
Windows Desktop: has their own keyboard, works
iPhone: keyboard activated only by text field

Mobile Chrome, Safari tested 

## 📸 Screenshots / Screen Recording

On slack accessibility feed-back

## 💡 Additional Context (Optional)

[Any console errors, network request failures, or theories on what broke?]

**Comment by PeekabooBear** [comment]
@m-ades @tasnimxahmed-cs 
Do we have any progress regarding this accessibility issue? I thought we were going to give an option to turn a regular keyboard on. I think this needs to be fixed as soon as possible.
